services/subscription.go

```diff
@@ -195,8 +231,9 @@ func receiveRunLog(le InitiatorSubscriptionLogEvent) {
 	runJob(le, data, le.Initiator)
 }
 
-// Parse the log and run the job specific to this initiator log event.
-func receiveEthLog(le InitiatorSubscriptionLogEvent) {
+// ReceiveEthLog parses the log and runs the job specific to this initiator log
+// event.
+func ReceiveEthLog(le InitiatorSubscriptionLogEvent) {
```

Comment:
Just for consistency, since `ReceiveRunLog` now needs to be public for [`subscription_test.go/TestStartRunLogSubscription_ValidateSenders`](https://github.com/smartcontractkit/chainlink/blob/feature/service-agreement-execution/services/subscription_test.go#L277).

Reply:
dafb3897